server/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -1465,25 +1452,13 @@ public void test_create_table_with_column_store_disabled() {
         }
     }
 
-    @Test
-    public void testCreateTableFailsIfNameConflictsWithView() {
```

Comment:
Is this restriction gone now?

Reply:
Oh forgot to comment on this one. We've an integration test case:

https://github.com/crate/crate/blob/ade08244b51b80dda130efe29fd9859887e6922f/server/src/test/java/io/crate/integrationtests/ViewsITest.java#L134-L145

Same reason as with the other analyzer test removal: The check happens later as it needs to be atomic in the cluster state update.
(We also had different error messages for the two cases..)